### PR TITLE
release-notes: fix syntax error

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

File: release-notes/stable.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 releases:
   39.20240322.3.1:
+    issues:
       - text: "Fix rpm-ostree CVE-2024-2905 (world-readable /etc/[g]shadow[-])"
         url: "https://github.com/coreos/fedora-coreos-tracker/issues/1705"
   39.20240322.3.0:
```
Comment (jlebon) [inline release-notes/stable.yml:6]: This one has the same issue.

File: release-notes/testing.yml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 releases:
   39.20240407.2.0:
+    issues:
       - text: "Fix rpm-ostree CVE-2024-2905 (world-readable /etc/[g]shadow[-])"
         url: "https://github.com/coreos/fedora-coreos-tracker/issues/1705"
   39.20240322.2.0:
```
Comment (jlebon) [inline release-notes/testing.yml:6]: This one has the same issue.

**Review by jlebon — COMMENTED** [review]
Man, we really need CI on this. :cry:

**Comment by dustymabe** [comment]
> Man, we really need CI on this.

yep

**Review by jlebon — APPROVED** [review]

**Review by travier — APPROVED** [review]

**Pull request merged**